main/reference/vstorage-temp.md

```diff
@@ -0,0 +1,144 @@
+# VStorage in Agoric
+
+In the Agoric platform, VStorage, or a Virtual Storage, is a key-value store that:
```

Comment:
We don't refer to it as "Virtual Storage" so no need to mention it.

Reply:
https://github.com/Agoric/agoric-sdk/tree/003f0c2232815a8d64a3f9a5b05521a10160ce34/golang/cosmos/x/vstorage#readme starts with "Virtual Storage"